guide/how-it-works/ecash/design-best-practices.md

```diff
@@ -0,0 +1,496 @@
+
+{% include tip/open.html color="green" icon="check" label="Avoid Jargon & NUTs 01-06" %}
+
+When displaying [NUTs](https://github.com/cashubtc/nuts) (Notation, Usage, and Terminology) that a mint supports, avoid simply listing NUT01, NUT02, etc. Instead, provide users with clear descriptions of each NUTs functionality. This helps users understand the features and capabilities of the mint at a glance. Since NUTs 01-06 are mandatory for Cashu protocol interoperability, focus on optional NUTs that add unique functionality. Consider using the descriptions provided in the official documentation to keep the information accurate.
+
+{% include tip/close.html %}
+
+#### Organizing Cashu metadata fields
+When designing your Cashu wallet, consider organizing these metadata fields in a logical, user-friendly manner to help users understand and interact with different mints. Here are some suggestions:
+
+- Use the `name` and short `description` in mint listings for quick identification of mints.
+- Display a detailed mint information page using `description_long`, `version`, and `contact` details.
+- Display the `motd` as a prominent notification when relevant to the specific mint.
+- Use the `nuts` information to create a features list that helps users understand each mint's capabilities.
+
+By effectively incorporating these Cashu metadata fields, you can create a more transparent and informative experience for users, helping them make informed decisions about which Cashu mints to trust and use. This is particularly important for Cashu, as users interact directly with individual mints rather than a federation structure like in Fedimint.
+
+### Edit mint URL & refresh mint settings
+
+{% include image-gallery.html pages = page.images_mint-url-edit %}
+
+It's useful to provide users with the ability to manually update or refresh mint settings in their wallets. While best practices suggest that wallets should auto-refresh and update mint settings periodically, there may be cases where this isn't supported, or a mint undergoes significant updates, such as a URL change. By allowing users to edit the mint URL or refresh settings manually, you ensure that users always have the latest mint configurations.
+
+### Pay to public key (P2PK)
+
+{% include image-gallery.html pages = page.images_p2pk %}
+
+[NUT11](https://github.com/cashubtc/nuts/blob/main/11.md) is a powerful feature that allows bitcoin backed ecash tokens to be securely locked to another user's public key, which is generated by the recipient's wallet. This ensures that only the intended recipient can redeem the ecash. NUT11 enables secure offline payments, preventing double-spending. Beyond these basics, NUT11 supports advanced use cases like timelocks and multisignature (multisig) setups, where ecash can be conditionally spent or jointly owned by multiple parties. When designing make sure these functionalities are clearly communicated to users, highlighting their practical benefits and flexibility.
```

Comment:
It might be worth noting that P2PK ecash is not derived from a seed phrase and thus cant be backed up via the seed phrase.

Reply:
Do you mean the actual ecash tokens that are locked with P2PK or the public key that a user generates?

Reply:
The actual proofs of P2PK or any nut10 secret are not derived from the secret so they are not backed up by the secret.

Reply:
This is important! I added a tip underneath this section: P2PK ecash tokens are not derived from a seed phrase. This means they cannot be backed up using a seed phrase, and in the event of a recovery from seed phrase the P2PK ecash tokens will be lost.